### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by vslotema**

### CODE REVIEW

**Comment by vslotema** [comment]
Source code of the lamp object.

File: app/src/main/java/com/example/aurora/PersonalThemeActivity.java [hunk 1]
```diff
@@ -77,61 +60,52 @@ public void getColorPicker() {
...
      * Code for the brightness bar which allows the user to increase or decrease the lamps brightness
      */
-
     void trackBrightnessBarProgress() {
         brightnessSeekBar.setOnSeekBarChangeListener(new SeekBar.OnSeekBarChangeListener() {
 
             @Override
             public void onProgressChanged(SeekBar seekBar, int progress, boolean fromUser) {
 
+                progress = progress + 4; //in order to avoid confusion with theme numbers which are sent to the raspberry pi
```
Comment (mircealungu) [inline app/src/main/java/com/example/aurora/PersonalThemeActivity.java]: Magic constants are bad for your code.

File: app/src/main/java/com/example/aurora/PersonalThemeActivity.java [hunk 2]
```diff
@@ -44,31 +38,20 @@ protected void onCreate(Bundle savedInstanceState) {
...
-        if (fragmentTheme == null) {
-            fragmentTheme = new ThemeFragment();
-            fm.beginTransaction()
-                    .add(R.id.fragment_theme, fragmentTheme)
-                    .commit();
-        }*/
+    @Override
+    protected void onPause() {
+        super.onPause();
+        (new Thread(new workerThread("2"))).start(); //THIS WILL STOP THE OCEAN LIGHTS
```
Comment (mircealungu) [inline app/src/main/java/com/example/aurora/PersonalThemeActivity.java]: 2 - magic constant

**Review by mircealungu — APPROVED** [review]

File: app/src/main/java/com/example/aurora/PersonalThemeActivity.java (hunk 2 above)
Comment (kubrauzunn) [reply app/src/main/java/com/example/aurora/PersonalThemeActivity.java]: The constant 2 here are used to send a message to the Raspberry pi. When the Raspberry pi receives a "2" is will stop playing the ocean theme. This is how we set up the communication between the pi and the application. Do you have suggestions on how we could improve it? It is working fine at the moment.

Comment (mircealungu) [reply app/src/main/java/com/example/aurora/PersonalThemeActivity.java]: Cool. Extract the 2 in a constant with a self-explanatory name. e.g. STOP_PLAYING_OCEAN_THEME = 2.

Comment (kubrauzunn) [reply app/src/main/java/com/example/aurora/PersonalThemeActivity.java]: Okay, thank you very much.

**Pull request merged**